### PR TITLE
[FIF-150] Rename project to Project Buzz

### PR DESCRIPTION
**Pull request by jleiva-gap**
Components renamed:
- Projects.
- Database.
- Documentation.
- Scripts.

### CODE REVIEW

**Comment by DavidJGapCR** [comment]
Student note service has a variable called FixItFridayStudentNotesRepository. Can we rename it?

**Comment by DavidJGapCR** [comment]
The last migration created, with name: 20200714162711-make-student-note-date-default-current-date still has the fif schema. We need to change it to buzz

**Comment by NSD-Xavier** [comment]
There is a new file EdFi.Buzz.Api/package-lock.json, but this is supposed to be run by yarn

**Comment by NSD-Xavier** [comment]
The EdFi.Buzz.Api\src\.env.example has BUZZ_API_HTTP_PORT = 3000 but EdFi.Buzz.Api\src\main.ts uses const httpPort = parseInt(process.env.BUZZ_API_PORT, 10);

File: EdFi.Buzz.Api/src/app.module.ts [hunk 1]
```diff
@@ -23,11 +23,11 @@ config({ path: `${__dirname}/.env` });
...
-      username: process.env.FIF_DB_USERNAME,
-      password: process.env.FIF_DB_PASSWORD,
-      database: process.env.FIF_DB_DATABASE,
+      host: process.env.BUZZ_API_DB_HOST,
+      port: parseInt(process.env.BUZZ_API_DB_PORT, 10),
+      username: process.env.BUZZ_API_DB_USERNAME,
+      password: process.env.BUZZ_API_DB_PASSWORD,
+      database: process.env.BUZZ_API_DB_DATABASE,
       entities: [`${__dirname}/**/*.entity.js`],
       synchronize: true
```
Comment (stephenfuqua) [inline EdFi.Buzz.Api/src/app.module.ts:32]: This line fails TeamCity build on the eslint step - should have a trailing comma.
```suggestion
      synchronize: true,
```

**Pull request merged**